### ISSUE TITLE
Warning red colour inconsistency with the status icons

### ISSUE THREAD

**Issue by dirtydancing** [numixproject/numix-icon-theme#286]
When looking into a couple of "warning red" status icons, I came across another colour inconsistency after the "light grey" colour inconsistency reported at https://github.com/numixproject/numix-icon-theme/issues/282

The "warning red" colour appears to be either <a href="http://www.color-hex.com/color/ef555c">#ef555c</a> or <a href="http://www.color-hex.com/color/ed535b">#ed535b</a>

Two examples for #ef555c

![syspeek-100](https://cloud.githubusercontent.com/assets/7164227/6574509/0a88d12c-c726-11e4-92d6-51401fa19323.png)

![battery-empty-symbolic](https://cloud.githubusercontent.com/assets/7164227/6574511/121c9cac-c726-11e4-9879-5b87f2dc13fa.png)

Two examples for #ed535b

![error](https://cloud.githubusercontent.com/assets/7164227/6574515/1f0f07e2-c726-11e4-8595-eab038fd7af1.png)

![xfce-system-lock](https://cloud.githubusercontent.com/assets/7164227/6574518/22ea7130-c726-11e4-845e-cbbdc9141806.png)

Maybe it would be a good idea to create a Numix (base) colour overview for quick reference and for consistency.

**Comment by dirtydancing** [comment]
@Foggalong Coming back to the idea above to create a Numix (base) colour overview: Taking the example from https://github.com/numixproject/numix-folders, I have been experimenting with that for the readme file of numix-icon-theme, cf. https://github.com/dirtydancing/numix-icon-theme/tree/colour-chart.

Screenshot:

![numix-icon-theme_readme-colours](https://cloud.githubusercontent.com/assets/7164227/6765262/8e3fcb78-cfd8-11e4-9d12-bf7d26de1914.png)

Does this make any sense at all, and should I create a pull request for this?

**Comment by wa4557** [comment]
Cool! 
But you're missing the "inactive" (the color of your audio status icon, if you mute your audio completely for example) color somehow. 

40%opacity #dcdcdc for light grey
and 
40% opacity #353535 for dark grey 

plus you could also add the color of the shadow (if there's one); 30% opacity black for all the colors in your post and 12% opacity black for all the "inactive" colors...

**Comment by dirtydancing** [comment]
@wa4557 Yeah! Good thinking! These would be useful additions. I just wanted to check with @Foggalong if I should create a pull request for this first. Or is it: create a pull request anyways, and take it from there? I am just getting somewhat more familiar with the GitHub workflow, and I was unsure how to proceed here.

**Comment by dirtydancing** [comment]
Pull request created at https://github.com/numixproject/numix-icon-theme/pull/321.

@wa4557 Thanks for your feedback so far, cf. above. Any more suggestions?

**Comment by wa4557** [comment]
@dirtydancing Nope; Looks good and everything's there that needs to be there, I hope this gets merged so we have an "officiall" way to iron out the last inconsistencies in the theme :)
